client/layout/guided-tours/docs/API.md

```diff
@@ -216,3 +216,53 @@ combineTours( {
   thirdTour: ThirdTour,
 } );
 ```
+
+## Targeting elements in Calypso
+
+In few places in GT you need to target a DOM elements in Calypso. There are currently two ways to do this:
+
+- `data-tip-target` attribute on the target element
+- using any css selector
+
+To find the DOM node, `document.querySelector` is used. By default, the query used is `[data-tip-target="${target}"]`. However, if `target` starts with a dot or a space character, it will be treated like a standard selector and be passed directly to the function, thereby allowing you to target elements that do no set a `data-tip-target` attribute.
+
+### [data-tip-target]
+
+You mark a target by adding `data-tip-target="some-name"` attribute to it with your value. Please be specific to avoid conflicts and use dash-case.
+
+In your tour, just put a value of your desired `[data-tip-target]` into the `target` prop.
+
+Example: you want to position a step of your tour to point to some input element.
```

Comment:
Realizing that we don't discuss the benefits of using `data-tip-target`. It almost feels like it's a legacy thing, when really there are advantages to it, or at least there are according to our premises:

- more unique than classes?
- less likely to be changed by someone else? (*)

Should we cover that? If so, what would you say about one method vs. the other?

(*): because it's a _weird_ attribute, it's also more greppable, leading developers to Guided Tours